editor.planx.uk/src/pages/FlowEditor/components/Settings/tests/ServiceSettings/PublicLinks.test.tsx

```diff
@@ -156,9 +157,9 @@ describe("A team without a subdomain has an offline, published service.", () =>
     });
 
     // Mocking window.location.origin
-    jest
-      .spyOn(window, "location", "get")
-      .mockReturnValue(mockWindowLocationObject);
+    vi.spyOn(window, "location", "get").mockReturnValue(
+      mockWindowLocationObject,
+    );
```

Comment:
nit: This might be something we can put in a `beforeEach()` (not for this PR, just a reminder to come back and take a look at this!)

Also getting a few linting issues in this file, will flag with @RODO94 tomorrow or open a quick PR here.

![image](https://github.com/user-attachments/assets/63355e61-4793-44c9-9d64-78362dc83b3a)